pkg/builder/interface.go

```diff
@@ -48,7 +48,7 @@ type Interface interface {
 	Builder() v1alpha1.BuildProvider
 
 	// Validate a Build for this flavor of builder.
-	Validate(*v1alpha1.Build, *v1alpha1.BuildTemplate) error
+	Validate(*v1alpha1.Build, v1alpha1.BuildTemplateInterface) error
```

Comment:
It looks like the template arg of this is never used, so we can probably remove it. Up to you whether you'd like to do that in this PR or I can do it after this is merged.

Reply:
removed the unused template